lib/px/utils/upload.lua

```diff
@@ -0,0 +1,334 @@
+-- Copyright (C) Yichun Zhang (agentzh)
```

Comment:
I think we should give the file a better name

Reply:
it's the original file name, I guess it's easier to keep track if the original file gets changed.

Reply:
I really worry, that a few years from now it will still be here and no one will know why this is the name.
We can add a comment with the real name and the origin

Reply:
renamed this file to "resty_upload.lua" and added comments with the original URL and PR.

Reply:
👑